### PR TITLE
Add documents to concat_mols

### PR DESCRIPTION
**Pull request by delta2323**
This PR adds a document to `concat_mols`

### CODE REVIEW

File: chainer_chemistry/dataset/converters.py [hunk 1]
```diff
@@ -2,7 +2,36 @@
...
+    Then, this function concatenates ``x`` 's into one array, and ``y`` 's
+    into another array, and returns a tuple of these two arrays. Another
+    example: consider each example is a dictionary of two entries whose keys
+    are ``'x'`` and ``'y'``, respectively, and values are arrays. Then, this
+    function concatenates ``x`` 's into one array, and ``y`` 's into another
+    array, and returns a dictionary with two entries ``x`` and ``y`` whose
+    values are the concatenated arrays.
+
+    When the arrays to concatenate have different shapes, the behavior depends
+    on the ``padding`` value. If ``padding`` is ``None`` (default), it raises
```
Comment (corochann) [inline chainer_chemistry/dataset/converters.py]: I guess you copied from 
https://github.com/chainer/chainer/blob/master/chainer/dataset/convert.py


`padding=0` is default, this sentence is confusing.

**Comment by corochann** [comment]
And if possible, I guess adding an example is more easier to understand the behavior of this function, as written in 
https://github.com/chainer/chainer/blob/master/chainer/dataset/convert.py

**Comment by delta2323** [comment]
OK, I'll do that.

**Comment by corochann** [comment]
Let me take over the remaining work.

**Pull request merged**